test/test_google_cloud_iam.py

```diff
@@ -0,0 +1,128 @@
+
+from cirrus.google_cloud.iam import (
+    GooglePolicyMember,
+    GooglePolicyRole,
+    GooglePolicyBinding,
+    GooglePolicy,
+)
+
+def test_create_google_policy():
+
+    role1 = GooglePolicyRole("roles/admin")
+    mem11 = GooglePolicyMember("user", "mem11@gmail.com")
```

Comment:
could be a personal thing, but I find the use of numbers here require more brain power than I want to comprehend what's going on. I'd find `member_a, member_b, ...` and `group_x` and `group_y` or something more helpful. At a minimum, some comments about what you're setting up and testing would be helpful

Reply:
I added comments to make this more clear